<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Instrument</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Temperature Sensor for Control</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="combo_SensorControl_Instrument"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="combo_SensorControl_Sensor"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Temperature Sensor for Measurements</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="combo_SensorMeas_Instrument"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="combo_SensorMeas_Sensor"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>stabilizing temperatures: Thresholds</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0,1,0,3,0">
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Temp [K]</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QDoubleSpinBox" name="spin_Sequence_threshold_slopeResiduals">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="spin_Sequence_threshold_stderr">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>slope residuals</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="spin_Sequence_threshold_T_K">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="spin_Sequence_threshold_slope_Kpmin">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Temp mean [K]</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>slope (relative) [1/min]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="spin_Sequence_threshold_Tmean_K">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="8">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>standard error (relative)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Load preset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushRefreshPresets">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QComboBox" name="combo_thresholdsLoadingPreset"/>
      </item>
      <item row="5" column="4" colspan="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Save preset</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4" colspan="2">
       <widget class="QLineEdit" name="lineEdit_thresholdsSavingPreset"/>
      </item>
      <item row="7" column="4" colspan="2">
       <widget class="QCommandLinkButton" name="command_sendThresholds">
        <property name="text">
         <string>Send Thresholds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_11">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Offset between VTI &amp; Sample [K]</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spin_Sequence_sampleOffsetLinear_K">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <double>300.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
